Hotfix for v2.2 for SSP data submission

ADPennington opened the pull request:
## Summary of Changes
Added filenames for UT and ME for SSP data submission for v2.2!.Thanks for the catch @ttran-hub 
![hotfix](https://user-images.githubusercontent.com/63075587/200845293-57ce5ed3-8d65-4cd7-a12c-49aa951ad634.PNG)


---

Evidence of missing SSP filenames in STTs model for UT (before fix):
![utah](https://user-images.githubusercontent.com/63075587/200697580-85c56844-ce34-467a-8aff-18541fbf2b01.PNG)


Evidence of SSP filenames in STTs model for UT (after fix):
![utah1](https://user-images.githubusercontent.com/63075587/200699125-dee04d9e-1e4a-4aa2-adca-b4aa0a7dc8c4.PNG)

ADPennington committed:
Hotfix for v2.2 for SSP data submission

added filenames for UT for SSP data submission for v2.2!

ADPennington commented:
@andrew-jameson @lilybl1 @jtimpe sorry for the short notice. but we noticed that the SSP filenames for UT are missing. cc: @ttran-hub

jtimpe reviewed (approved):
Looks good to me. in addition to what alex posted, i ran a couple data files through (first ssp then regular)

<img width="1628" alt="image" src="https://user-images.githubusercontent.com/111305129/200712917-a8d1eff0-8736-4b20-94f7-4ff30dec2e19.png">

<img width="1014" alt="image" src="https://user-images.githubusercontent.com/111305129/200735377-5ededf19-030b-4ad5-86ef-be4758d0b01a.png">

ADPennington committed:
ME also missing SSP filenames

audited `states.csv` to confirm all SSP states had filenames. only UT and ME were missing these.

ADPennington committed:
add space for ME SSP files

ADPennington committed:
close quotes for ME SSP Section3

ADPennington commented:
also tested for maine (tanf and ssp)

_model_
![hotfix3](https://user-images.githubusercontent.com/63075587/200856699-ade8c329-372f-489f-a9f2-53f0dde9c056.PNG)


_titan_
![hotfix2](https://user-images.githubusercontent.com/63075587/200856719-2dfdb9e4-d328-41b5-8a60-f75c3294d1d2.PNG)
